scripts/install_mooc_nbs.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/env bash
+
+
+echo "######################"
+echo "COPYING MOOC Notebooks"
+echo "######################"
+
+git clone https://github.com/CCI-Tools/cate-edu
+cd cate-edu || exit
+git checkout alicja-xxx-update_of_nbs
+
+cp -r ECVs/ ~/examples
```

Comment:
This needs to be fixed. Because we are looking at the mooc nbs, right?

Reply:
Correct.

Reply:
Can you fix it?